packages/flutter/lib/src/cupertino/date_picker.dart

```
@@ -435,6 +443,12 @@ class CupertinoDatePicker extends StatefulWidget {
   /// Defaults to false.
   final bool showDayOfWeek;
 
+  /// Whether to show the time separator between hour and minute in
+  /// [CupertinoDatePickerMode.time] and [CupertinoDatePickerMode.dateAndTime] mode.
+  ///
+  /// Defaults to false.
```

We should mention that this will throw an error in unsupported modes.